src/Repository/ProductRepository.php

```diff
@@ -252,4 +252,42 @@ public function findProductByProductReference($carrierReference)
 
         return $this->db->getRow($query) ?: null;
     }
+
+    /**
+     * @param int $carrierReference
+     * @param int $countryId
+     *
+     * @return array|bool|mysqli_result|PDOStatement|resource|null
+     * @throws PrestaShopDatabaseException
+     */
+    public function checkIfCarrierIsAvailableInCountry(int $carrierReference, int $countryId)
```

Comment:
@GytisZum check if argument types are available to PHP 5.6

Reply:
![image](https://github.com/Invertus/dpdbaltics17/assets/17498592/eab43808-e8be-4d82-829c-5350940e0d6e)
Composer defines that it should support PHP 5.6 though argument types are not supported for this version of PHP. When working on this module you can change PHPStorm version to 5.6 or discuss if you still need to support it.